.drone.yml

```diff
@@ -53,7 +53,7 @@ steps:
   environment:
     BUILDKIT_HOST: tcp://buildkitd.ci.svc:1234
   commands:
-    - make
+    - make -j
```

Comment:
is buildkit able to track deps for concurrent builds correctly?

Reply:
That is what they claim.

Reply:
That is what they claim.

Reply:
This was only for testing. I have removed this from this PR.